oshi-core/src/main/java/oshi/software/os/OperatingSystem.java

```diff
@@ -91,6 +92,17 @@
      *         process id if it is running; null otherwise
      */
     OSProcess getProcess(int pid);
+    
+    /**
+     * Gets information on a currently running processes. This was primarily written
+     * to provide an optimized mechanism for windows based operating systems.
+     *
+     * @param pids
+     *            A collection of process IDs
+     * @return An {@link oshi.software.os.OSProcess} object for the specified
+     *         process ids if it is running
+     */
+    Collection<OSProcess> getProcesses(Collection<Integer> pids);

```

Comment:
I don't mind using Collection as a parameter but I'm not sure I like it as a return type rather than Set or List -- which we use most other places in the code.